pkg/cnab/config-adapter/adapter.go

```diff
@@ -307,6 +307,12 @@ func (c *ManifestConverter) generateDependencies() *extensions.Dependencies {
 	}
 
 	for _, dep := range c.Manifest.Dependencies {
+		// Use the deprecated Tag value if Reference not set
+		// We should remove this field completely before 1.0
+		if dep.Tag != "" && dep.Reference == "" {
```

Comment:
I would have expected that when we load the porter.yaml with tag set for a dependency, that dep.Reference would be populated with the tag value (the backwards compatibility fix). Are you sure this is needed?

I don't think it's hurting anything but I'm wondering if I am missing a code path when thinking about this fix.

Reply:
Great point -- indeed, we needed to update the Dependencies field to be a slice of pointers so that the `Validate` logic in manifest.go would persist.  Updated.